### ISSUE TITLE
Requests cannot be inserted into folders.

### ISSUE THREAD

**Issue by rangicus** [getinsomnia/insomnia#5985]
### Expected Behavior

When clicking and dragging a request into a folder, I expect a dashed-line to appear, showing where the request will go.

When releasing left click while the request is in the folder, I expect the request to go into the folder.

### Actual Behavior

When clicking and dragging a request into a folder, a dashed line appears **outside** of the folder.

When releasing left click while the request is in the folder, the request lands outside of the folder.

### Reproduction Steps

1. Create a folder.
2. Create a request.
3. Left-click and drag the request into the folder.

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Additional Information

This issue already exists (#1486), but it was closed without being resolved, and I don't have the power to reopen it.

Version: Insomnia 2023.2.2
Build date: 5/17/2023
OS: Windows_NT x64 10.0.22621
Electron: 22.0.0
Node: 16.17.1
V8: 10.8.168.20-electron.0
Architecture: x64

### Insomnia Version

2023.2.2

### What operating system are you using?

Windows

### Operating System Version

Windows_NT x64 10.0.22621

### Installation method

Installer

**Comment by luke-schleicher** [comment]
I recently updated Insomnia and am now experiencing this issue as well. Super annoying

Version: Insomnia 2023.1.0
Build date: 3/9/2023
OS: Darwin x64 21.5.0
Electron: 22.0.0
Node: 16.17.1
V8: 10.8.168.20-electron.0
Architecture: x64

**Comment by TheDevMinerTV** [comment]
Same issue here.

```
Version: Insomnia 2023.2.2
Build date: 5/17/2023
OS: Windows_NT x64 10.0.19045
Electron: 22.0.0
Node: 16.17.1
V8: 10.8.168.20-electron.0
Architecture: x64
```

**Comment by MarkCay** [comment]
Same issue here. 


```
Device: macOs 13.1 (22C65)

Version: Insomnia 2022.7.5
Build date: 1/19/2023
OS: Darwin arm64 22.2.0
Electron: 22.0.0
Node: 16.17.1
V8: 10.8.168.20-electron.0
Architecture: arm64
```

**Comment by Maritims** [comment]
#1486 should be reopened and merged with this issue.

**Comment by gian1200** [comment]
Still an issue on Insomnia 2023.5.5

![image](https://github.com/Kong/insomnia/assets/405347/2f22108d-0248-46ae-afbb-6113588f38c4)

Although it feels more like a visual desync issue. + a visual hint issue. 

If I drag a request from folder to outside, the list view doesn't change. However, when opening/closing any folder on the list, the view gets updated and the movement is now applied.

If I drag a request from outside to folder, the visual drag hint doesn't correctly tells you that the movement is going to be inside the folder. When completing the dragging, the request seems to be still on the original location. However, when opening/closing any folder, the movement is now applied.

It seems as if closing/opening any folder makes the view get refreshed with the current (correct) state.

**Comment by jackkav** [comment]
fixed in #6323
